src/openms/source/SYSTEM/ExternalProcess.cpp

```diff
@@ -81,6 +81,10 @@ namespace OpenMS
 
   ExternalProcess::RETURNSTATE ExternalProcess::run(const QString& exe, const QStringList& args, const QString& working_dir, const bool verbose, String& error_msg, IO_MODE io_mode)
   {
+    // pass environment variables to child process
+    QProcessEnvironment env = QProcessEnvironment::systemEnvironment();
+    qp_->setProcessEnvironment(env);
```

Comment:
why is this required?
According to https://doc.qt.io/qt-5/qprocess.html#processEnvironment,
the calling process' environment is the default.

Reply:
hmm in my case:

I ran a tool via a bash script.
- In the bash script I export the OPENMS_DATA_PATH
- the tool executed in the script starts an ExternalProcess and executes a python script
Somehow the OPENMS_DATA_PATH was not set in the python script

After this change it worked (I may double check but I did not get the error message again)

Reply:
Hmm after reading the docs I agree with @cbielow that this should make no difference.

Reply:
k thanks for checking. I will need to dig into this a bit more